folding-schemes/src/folding/hypernova/decider_eth.rs

```diff
@@ -223,11 +224,13 @@ pub mod tests {
     use ark_bn254::{constraints::GVar, Bn254, Fr, G1Projective as Projective};
     use ark_groth16::Groth16;
     use ark_grumpkin::{constraints::GVar as GVar2, Projective as Projective2};
-    use std::time::Instant;
+    use ark_serialize::{Compress, Validate};
 
     use super::*;
     use crate::commitment::{kzg::KZG, pedersen::Pedersen};
-    use crate::folding::hypernova::PreprocessorParam;
+    use crate::folding::hypernova::cccs::CCCS;
+    use crate::folding::hypernova::{PreprocessorParam, ProverParams, VerifierParams};
+    use crate::folding::nova::decider_eth::VerifierParam;
```

Comment:
it's easy to get confused between `VerifierParams` and `VeriferParam`, the first being the HyperNova IVC verifier params, and the second being the HyperNova's DeciderETH verifier params.
Maybe something like
```suggestion
    use crate::folding::hypernova::{PreprocessorParam, ProverParams, VerifierParams as HyperNovaVerifierParams};
    use crate::folding::nova::decider_eth::VerifierParam;
```
(where `decider_eth::VerifierParam` is not renamed since is the Decider's param and we're at the Decider file.

Reply:
Good suggestion, this will indeed surely reduce confusion. [67fcb0f](https://github.com/privacy-scaling-explorations/sonobe/pull/159/commits/67fcb0faec6367ce91922a83fe00660b933a4dc9)